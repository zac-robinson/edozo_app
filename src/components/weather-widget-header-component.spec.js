import React from 'react';
import { shallow } from 'enzyme';

import WeatherWidgetHeader from './weather-widget-header-component';

describe('WeatherWidgetHeader component', () => {
  it('should render correctly', () => {
    const props = {
      city: 'York',
      temperature: {
        temp: '45'
      },
      wind: {
        speed: 3444
      },
      conditions: {
        icon: 'sun'
      }
    };
    const component = shallow(<WeatherWidgetHeader {...props} />);

    expect(component).toMatchSnapshot();
  });
});
