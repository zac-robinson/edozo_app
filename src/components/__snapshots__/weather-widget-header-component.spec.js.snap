// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherWidgetHeader component should render correctly 1`] = `
<styled.div>
  <City
    city="York"
  />
  <Temperature
    temperature={
      Object {
        "temp": "45",
      }
    }
  />
  <Wind
    wind={
      Object {
        "speed": 3444,
      }
    }
  />
  <Conditions
    conditions={
      Object {
        "icon": "sun",
      }
    }
  />
</styled.div>
`;
